example/lib/src/features/home/views/pages/home.dart

```diff
@@ -35,7 +35,7 @@ class _HomeScreenState extends State<HomeScreen> {
   Widget build(BuildContext context) {
     return ToastificationConfigProvider(
       config: ToastificationConfig(
-        marginBuilder: (alignment) => const EdgeInsets.fromLTRB(0, 16, 0, 110),
+        marginBuilder: (context, alignment) => const EdgeInsets.fromLTRB(0, 16, 0, 110),
```

Comment:
Nice improvement 

Luckily, the overlay widget is updating by Observing on Media query widget